dist/images/start-ipsec.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+set -euo pipefail
+
+podNames=`kubectl get pod -n kube-system -l app=ovs -o 'jsonpath={.items[*].metadata.name}'`
+for pod in $podNames; do
+  caPod=$pod
+  break
+done
+
+echo " initing CA $caPod "
```

Comment:
The first letter in output should be capitalized. All the `echo` should be modified

Reply:
got it